packages/cli/src/telemetry/index.ts

```diff
@@ -86,7 +86,7 @@ async function checkOptIn(interactive: boolean): Promise<GlobalTelemetryOptions
     await fs.writeJson(globalDataPath, globalOptions);
   }
 
-  if (localOptIn === undefined && globalOptions !== undefined) {
+  if (project.exists() && localOptIn === undefined && globalOptions !== undefined) {
```

Comment:
Does this mean that a user will be asked repeatedly if they want to enable telemetry when `init` was not run (e.g. when only using `compile`)?

Reply:
No, they will only be asked once, and this global setting will be saved and reused. The change here causes that we only store this information as a local setting once there is an existing project file.